lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -192,12 +181,24 @@ TEST_F(SessionManagerHandlerTest, test_SetAmfSessionContext) {
       &server_context, &request,
       [this](grpc::Status status, SmContextVoid Void) {});
 
+  assert_sm_session_context(&request);
```

Comment:
why is this assert check here instead of being checked right after `set_sm_session_context`

Reply:
Though we are setting GRPC message `SetSMSessionContext` request locally via `set_sm_session_context` , we can assume the flow like, once AMF fills the `SetSMSessionContext` request and sends to sessiond via GRPC call `SetAmfSessionContext` and then we can assert via `assert_sm_session_context` function. I thought it would be more readable. Please suggest me if it is fine or not.